packages/core/admin/admin/src/components/PluginsInitializer.tsx

```diff
@@ -90,17 +87,16 @@ type SetPluginReadyAction = {
 
 type Action = SetPluginReadyAction;
 
-const reducer: React.Reducer<State, Action> = (state = initialState, action: Action): State =>
-  produce(state, (draftState) => {
-    switch (action.type) {
-      case 'SET_PLUGIN_READY': {
-        set(draftState, ['plugins', action.pluginId, 'isReady'], true);
-        break;
-      }
-      default:
-        return draftState;
+const reducer: React.Reducer<State, Action> = (state = initialState, action: Action): State => {
```

Comment:
I think it could be good to leave a comment like:

> immer produce is intentionally not used in this reducer since the Plugin class is not "immerable". See: https://immerjs.github.io/immer/complex-objects/